ray-operator/controllers/ray/utils/validation.go

```diff
@@ -164,3 +135,47 @@ func ValidateRayServiceSpec(rayService *rayv1.RayService) error {
 	}
 	return nil
 }
+
+func ValidateGCSFaultTolerance(spec *rayv1.RayClusterSpec, annotations map[string]string) error {
```

Comment:
We can do the refactoring, but it's better to do it in a follow-up PR. Let's focus on calling `ValidateRayClusterSpec` in RayJob controller in this PR.

Reply:
This function has been removed and the code has put back inside `ValidateRayClusterSpec`.